lib/oli_web/router.ex

```diff
@@ -417,7 +417,15 @@ defmodule OliWeb.Router do
     live("/:project_id/objectives", ObjectivesLive.Objectives)
 
     # Experiment management
-    live("/:project_id/experiments", Experiments.ExperimentsView)
+
+    live_session :another_load_projects,
+      on_mount: [
+        OliWeb.LiveSessionPlugs.SetUser,
+        OliWeb.LiveSessionPlugs.SetProject
+      ] do
+      live("/:project_id/experiments", Experiments.ExperimentsView)
+    end
```

Comment:
Do we plan to expand other routes in this `live_session`? If not, perhaps we could define the `on_mount` within the Experiments liveview.
If we are going to expand it, what do you think about another name for the live session rather than "another_load_projects"? Maybe just "experiments"?

Reply:
Nice suggestion! I learned something today! Thank!